cli/js/compiler.ts

```diff
@@ -12,7 +12,7 @@
 // NOTE: this import has side effects!
 import "./ts_global.d.ts";
 
-import { bold, cyan, yellow } from "./colors.ts";
+import { bold, cyan, yellow } from "../../std/fmt/colors.ts";
```

Comment:
Hmm... I think we shouldn't do this (because std/fmt/colors.ts isn't part of the cli crate). I'm actually surprised this even works.

Reply:
@kitsonk wanted to do that, actually.

Reply:
Ideally we could do this, but I'd rather explore this possibility in its own standalone PR. In your changes, let's just worry about improving the colors.

Reply:
I guess yes, this is likely what was causing pointer magic for the global enable variable.
I'd have to move it into the cli crate and let it be a duplicate in fmt.